src/main/java/org/dita/dost/module/BranchFilterModule.java

```diff
@@ -374,10 +375,6 @@ private void generateCopies(final Element topicref, final List<FilterUtils> filt
                 writer.setCurrentFile(dstAbsUri);
                 final List<XMLFilter> pipe = singletonList(writer);
 
-                final File dstDirUri = new File(dstAbsUri.resolve("."));
```

Comment:
No need to create directories into temporary directory in disk, the `Store` will handle that when resource is created.